src/rpft/parsers/sheets.py

```diff
@@ -9,55 +10,79 @@
 from googleapiclient.discovery import build
 
 
-class CSVSheetReader:
+class AbstractSheetReader(ABC):
+    def csv_path(self):
+        return None
+
+    @abstractmethod
+    def get_main_sheets(self):
+        pass
```

Comment:
The idea of getting main sheets doesn't sit well with me because I think it breaks the sheet reader abstraction. A single spreadsheet has only one main sheet - the content index. Should the main sheet for a composite reader be a merger/concatenation of all the content index sheets?

Reply:
Conceptually, there's no reason why there couldn't be multiple main sheets. For example, we could adopt a naming convention where all sheets with the prefix `content_index` are main sheets, or we can add multiple main sheets to a CSVReader.